src/components/Map/StandardMap.vue

```diff
@@ -75,6 +76,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    centerUrl: {
```

Comment:
I'd call those props (including the parent ones) `forceCenter` and `forceZoom` to make their behavior clearer. They don't carry a URL in them, after all.